src/index.js

```diff
@@ -182,6 +182,9 @@ class KadDHT extends EventEmitter {
     this.contentFetching = contentFetching(this)
     this.contentRouting = contentRouting(this)
     this.peerRouting = peerRouting(this)
+
+    this.onPut = (record, peerId) => {}
```

Comment:
These should be provided via the constructor with a default to an empty function.

```js
const dht = new DHT({
  libp2p,
  // ... others
  onPut: (record, peerId) => {}, // User provided function
  onRemove: (record) => {} // User provided function
})
```

```js
  /**
   * Create a new KadDHT.
   * @param {Object} props
   * @param {Libp2p} [props.libp2p] the libp2p instance
   * @param {(record: Record, peerId: PeerId) => void} [onPut = () => {}]
   * @param {(record: Record) => void} [onRemove = () => {}]
   * ...
   */
   constructor({
     // ...
    onPut = () => {}
   }) {
     this.onPut = onPut
   }
```